admin/types.d.ts

```diff
@@ -0,0 +1,41 @@
+export type ReticulumMetaT = {
```

Comment:
@bryanenders @netpro2k Was hoping to get some insight on if any of these types are conditional so I can throw the "?" on them or if we think they are all expected every time?